<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>C-NOVA Panorama</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="leftLayout">
      <item>
       <layout class="QHBoxLayout" name="topRow">
        <item>
         <widget class="QPushButton" name="btnLoad">
          <property name="text">
           <string>Загрузить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSave">
          <property name="text">
           <string>Сохранить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnAddRow">
          <property name="text">
           <string>Добавить строку</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnRemoveRow">
          <property name="text">
           <string>Удалить строку</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="headerRow">
        <item>
         <widget class="QLineEdit" name="lineMachine">
          <property name="placeholderText">
           <string>Номер комплекса</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineDate">
          <property name="placeholderText">
           <string>Дата (дд.ММ.гггг)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineTime">
          <property name="placeholderText">
           <string>Время (чч:мм:сс.мс)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
